topic4/contracts/Math.sol

```diff
@@ -3,5 +3,48 @@ pragma solidity ^0.4.23;
 import './interfaces/IMath.sol';
 
 contract Math is IMath {
-		
+
+    uint _stateNumber = 0;
+
+	function stateNumber() external view returns (uint) {
```

Comment:
4. Keep in mind that if you have public function and public state variable with the same name, the function will shadow the state variable. (related to **3.**)